app/javascript/app/components/compare-ndc-content-overview/compare-ndc-content-overview-styles.scss

```diff
@@ -1,26 +1,26 @@
 @import '~styles/layout.scss';
 
 .col6 {
-  @extend %grid;
+  @include columns();
 
-  margin-bottom: 10px;
+  margin-bottom: $gutter-padding;
 
   @media #{$tablet-landscape} {
-    @include msGridColumns(1fr, 1fr, 1fr, 1fr, 1fr, 1fr);
+    @include columns((6, 2, 1, 3));
```

Comment:
I don't really get why we have so many columns here if we only have two elements